documentation/schema.json

```diff
@@ -1166,9 +1166,37 @@
           "type": "string",
           "description": "The name of the egress provider to which the logs are egressed.",
           "minLength": 1
+        },
+        "Format": {
+          "description": "The LogFormat used when egressing logs.",
+          "default": "EventStream",
+          "oneOf": [
+            {
+              "type": "null"
+            },
+            {
+              "$ref": "#/definitions/LogFormat"
+            }
+          ]
         }
       }
     },
+    "LogFormat": {
+      "type": "string",
+      "description": "",
+      "x-enumNames": [
+        "None",
+        "NDJson",
+        "EventStream",
+        "JsonSequence"
+      ],
+      "enum": [
+        "None",
```

Comment:
This value should not be an acceptable value in the configuration. Please remove this from the enum and update the existing code to use `LogFormat?` with the understanding that when it is null to treat the same way as the prior usage of the `None` value.